labelledslider.cpp

```diff
@@ -36,13 +36,17 @@ int LabelledSlider::value() const {
 }
 
 // public slot
-void LabelledSlider::setValue(int v) {
-  slider->setValue(v);
+void LabelledSlider::setValue(double v) {
+  preciseValue = v;
+  slider->setValue((int)(v + 0.5));
```

Comment:
I like this more that the other places where rounding is done.
Here an explicit cast to int is visible for a reviewer, so intended rounding could be expected.

Reply:
Is there a specific reason for avoiding C++style cast here?
`slider->setValue(static_cast<int>(v + 0.5));`

Reply:
slider->setValue(static_cast<int>floor(v + 0.5));

I just realized, that the DepthSlider can now be also negative . . .